### PR TITLE
fix(server): Change clientLogLevel order to be called first

### PR DESCRIPTION
**Pull request by hiroppy**
- [x] This is a **bugfix**
- [ ] This is a **feature**
- [ ] This is a **code refactor**
- [ ] This is a **test update**
- [ ] This is a **docs update**
- [ ] This is a **metadata update**

### For Bugs and Features; did you add new tests?

yes

### Motivation / Use-Case

maybe related #1972.

## before

Screen Shot 2019-06-05 at 16 18 45

## after

Screen Shot 2019-06-05 at 16 21 26

### Breaking Changes

no

### CODE REVIEW

File: lib/Server.js [hunk 1]
```diff
@@ -694,11 +694,15 @@ class Server {
...
+      if (this.clientLogLevel) {
+        this.sockWrite([connection], 'log-level', this.clientLogLevel);
+      }
+
       if (this.hot) {
         this.sockWrite([connection], 'hot');
       }
 
-      if (this.options.liveReload !== false) {
+      if (this.options.liveReload) {
```
Comment (evilebottnawi) [inline lib/Server.js]: Can be breaking change for non boolean value, let's avoid this in this PR

Comment (hiroppy) [reply lib/Server.js]: ok, I'll change it at next major version.

Comment (evilebottnawi) [reply lib/Server.js]: Yes

**Comment by hiroppy** [comment]
Sorry, I contained another commit to put .gitkeep.

**Pull request merged**